File: day40/code.py
#Leetcode Solution

# class Solution:
#     def sortColors(self, nums: List[int]) -> None:
#         nums.sort()


#An altered solution for dry run.

def sortColors(nums):
    nums.sort()
    
n=int(input("Enter length of list: "))
nums=[]
for i in range(0,n):
    nums.append(int(input("Enter value: ")))

result=sortColors(nums)
print(result)

File: day43/code.cpp
// Leetcode Solution

// class Solution {
// public:
//     int searchInsert(vector<int>& nums, int target) {
//         int n = size(nums);
//         if (target>nums[n-1]){
//             return n;
//         }
//         for (int i=0;i<n;i++){
//             if (nums[i]==target){
//                 return i;
//             }
//             else if (nums[i]>target){
//                 return i;
//             }
//         }
//         return 0;
//     }
// };